ext/settings.html

```diff
@@ -1581,6 +1581,34 @@ <h1>Yomichan Settings</h1>
                 <button class="icon-button"><span class="icon-button-inner"><span class="icon" data-icon="material-right-arrow"></span></span></button>
             </div>
         </div></div>
+        <div class="settings-item advanced-only">
+            <div class="settings-item-inner">
+                <div class="settings-item-left">
+                    <div class="settings-item-label">
+                        Show card tags
+                        <a class="more-toggle more-only" data-parent-distance="4">(?)</a>
+                    </div>
+                </div>
+                <div class="settings-item-right">
+                    <select data-setting="anki.displayTags">
+                        <option value="never">Never</option>
+                        <option value="always">Always</option>
+                        <option value="non-standard">Non-Standard</option>
+                    </select>
+                </div>
+            </div>
+            <div class="settings-item-children more" hidden>
+                <p>
+                    When coming across a word that is already in an anki deck, show an icon that lets you quickly see
```

Comment:
Some slight wording tweaks:

* > When coming across a word that is already in an anki deck, show an icon that lets you quickly see which tags that card has.

   to 
   
  > When coming across a word that is already present in an Anki deck, a button will appear that shows the tags the card has.
  
  Rationale:
  * Anki capitalized
  * Slight rewording to omit the word "you", which is avoided for most descriptions.

<br>

* > If set to &lt;em&gt;Non-Standard&lt;/em&gt; then all tags ...

  to
  
  > If set to &lt;em&gt;Non-Standard&lt;/em&gt;, all tags ...
  
  Rationale: comma should be there anyway, but removing "then" results in less repetition with the following sentence.